<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LassoClass</class>
 <widget class="QMainWindow" name="LassoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1617</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lasso</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>1080</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1080</width>
      <height>1080</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>400</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1617</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>FILE</string>
    </property>
    <addaction name="actionINPUT"/>
    <addaction name="actionSEARCH"/>
   </widget>
   <addaction name="menuFILE"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionINPUT">
   <property name="text">
    <string>INPUT</string>
   </property>
  </action>
  <action name="actionSEARCH">
   <property name="text">
    <string>SEARCH</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Lasso.qrc"/>
 </resources>
 <connections/>
</ui>
